src/pybindings/Bnder_Code_Gen.md

```diff
@@ -0,0 +1,68 @@
+# Generating PyBind with Binder
+
+## Install Binder
+
+-   Installation instructions can be found
+    [here](https://cppbinder.readthedocs.io/en/latest/install.html).
+
+## Binder Command
+
+binder --config `./CHIPDeviceController.config` --suppress-errors --root-module
```

Comment:
Could this binder stuff be dropped from this PR? We don't have consensus about using this tool.